lib-gphotos/client.go

```diff
@@ -57,73 +56,141 @@ func NewClient(oauthHTTPClient *http.Client, maybeToken ...*oauth2.Token) (*Clie
 }
 
 // GetUploadToken sends the media and returns the UploadToken.
-func (client *Client) GetUploadToken(r io.Reader, filename string) (token string, err error) {
-	req, err := http.NewRequest("POST", fmt.Sprintf("%s/%s/uploads", basePath, apiVersion), r)
+func (c *Client) GetUploadToken(r io.ReadSeeker, filename, uploadURL string, fileSize int64) (string, string, error) {
```

Comment:
We should put what return variables are. You can give it a name or use function documentation block